.github/workflows/detect-breaking-changes-report.yml

```diff
@@ -15,6 +15,10 @@ jobs:
     env:
       ARTIFACT_NAME: 'levitate' # The name of the artifact that we would like to download
       ARTIFACT_FOLDER: '${{ github.workspace }}/tmp' # The name of the folder where we will download the artifact to
+    permissions:
+      contents: write
```

Comment:
I don't think this action requires `contents: write`? I think it can get away with `contents: read` @grafana/plugins-platform-backend